recordlinkage/index.py

```diff
@@ -459,6 +459,175 @@ def _dedup_index(self, df_a):
             levels=levels, labels=labels, verify_integrity=False)
 
 
+class NeighbourhoodBlock(Block):
```

Comment:
Move this class to a new Python file (for example neighbourhoodblock.py) in the contrib folder https://github.com/J535D165/recordlinkage/tree/master/recordlinkage/contrib/index.

Reply:
done